rfcs/text/0016-target-process.md

```diff
@@ -65,10 +134,13 @@ The goal here is to research and understand the impact of these changes on users
 
 ## Concerns
 
+
 <!--
 Stage 1: Identify potential concerns, implementation challenges, or complexity. Spend some time on this. Play devil's advocate. Try to identify the sort of non-obvious challenges that tend to surface later. The goal here is to surface risks early, allow everyone the time to work through them, and ultimately document resolution for posterity's sake.
 -->
 
+The biggest concern is the duplication of fields and the double-nested `process` group at `process.target.parent`. This could require some updates to our reuse mechanism, but that's an issue internal to this repository. We should make sure that we don't accidentally populate `process.parent.target`, which would have different meaning. Because of this, we will need to make sure that we articulate what each reuse means, similar to https://www.elastic.co/guide/en/ecs/current/ecs-user.html#ecs-user-nestings.
```

Comment:
Thanks for calling this out.

Changes will need to be made to support reusing `process.*` fields as `parent.*` underneath `process.target` since self-nestings (fields reused within themselves) aren't carried around to other places (avoids nestings like `user.target.*` from getting carried over to `source.user.target.*`)